go-controller/pkg/ovn/egressgw.go

```diff
@@ -696,42 +712,104 @@ func (oc *Controller) addHybridRoutePolicyForPod(podIP net.IP, node string) erro
 	return nil
 }
 
-// delHybridRoutePolicyForPod handles deleting a higher priority allow policy to allow traffic to be routed normally
-// by ecmp routes
+// delHybridRoutePolicyForPod handles deleting a logical route policy that
+// forces pod egress traffic to be rerouted to a gateway router for local gateway mode.
 // force is used to indicate to force remove LRPs, useful for when going from LGW->SGW
 func (oc *Controller) delHybridRoutePolicyForPod(podIP net.IP, node string, force bool) error {
```

Comment:
i'm thinking we should remove the force bool and make it a separate function to do the delAllHybridRoutePolicies. Wdyt?